traitsui/testing/tester/ui_wrapper.py

```diff
@@ -73,6 +73,44 @@ def __init__(self, target, registries, delay=0):
         self._registries = registries
         self.delay = delay
 
+    def help(self):
+        """ Print help messages.
+        (This function is intended for interactive use.)
+        """
+        interactions = dict()
+        locations = dict()
+        for registry in self._registries[::-1]:
+            for type_ in registry.get_interactions(self.target.__class__):
+                interactions[type_] = registry.get_interaction_doc(
+                    self.target.__class__, type_)
+
+            for type_ in registry.get_locations(self.target.__class__):
+                locations[type_] = registry.get_location_doc(
+                    self.target.__class__, type_)
+
+        print("Interactions")
+        print("------------")
+        pairs = sorted(interactions.items(), key=lambda value: repr(value[0]))
+        for interaction_class, doc in pairs:
+            print(repr(interaction_class))
+            print(
+                *("    " + line for line in doc.split("\n")),
```

Comment:
If I understand correctly, the `"          "` adds indentation to the help documentation. IMO it is cleaner and more readable to use `textwrap.indent` instead and replace `"        "` with `r"\t"` (maybe)